charts/pega/charts/hazelcast/values.yaml

```diff
@@ -1,11 +1,25 @@
 ---
 image: "YOUR_HAZELCAST_IMAGE:TAG"
+clusteringServiceImage: "YOUR_CLUSTERING_SERVICE_IMAGE:TAG"
 imagePullPolicy: "Always"
 # Enter the number of initial members in Hazelcast cluster.
 replicas: 3
 # Setting below to true will deploy the infinity in client-server Hazelcast model.
 # Note: Make sure to set this value as "false" in case of platform version below "8.6". If not set this will fail the installation.
 enabled: true
+clusteringServiceEnabled: false
+
+# Setting related to Hazelcast migration.
+migration:
+  # Setting up to initiate the migration job.(Only for CMC/CloudK have InfinityOps service for migration task.)
```

Comment:
Lets not talk about Cloudk stuff and its internals in the public facing repository.

Reply:
Setting to initiate a migration from embedded to client-server model. This is not required if you are running in a client-server model and you are updating your Hazelcast service.

Reply:
@taz-pega-work , yes 'embeddedToCSMigration' flag is only for embedded to c/s hazelcast migration process only.

Reply:
@taz-pega-work , this is not just for embedded to c/s hazelcast migration. It is to migrate from old hazelcast to new hazelcast without any downtime on running environment. It can be embedded to c/s or c/s to c/s hazelcast migration.

Reply:
@kishorv10 , removed the cloudk ref from the comment